pydantic/schema.py

```diff
@@ -277,6 +277,8 @@ def get_field_schema_validations(field: ModelField) -> Dict[str, Any]:
                 f_schema[keyword] = attr
     if field.field_info is not None and field.field_info.const:
         f_schema['const'] = field.default
+    if field.field_info is not None and field.field_info.read_only:
```

Comment:
This is incorrect I think, as per JSON Schema this suggests the field cannot be set when creating a model (e.g. like `POST`ing to the endpoint who's `OPTIONS` endpoint returns this schema)

Reply:
Admittedly, I didn't put much thought into the JSON schema as it's not something I currently plan to use.  I see that you are correct here and will revert the commit that adds readOnly to the json schema.